UnityProject/Assets/Scripts/Objects/Disposals/DisposalBin.cs

```diff
@@ -233,10 +233,18 @@ public override void ServerPerformInteraction(PositionalHandApply interaction)
 		public bool WillInteract(MouseDrop interaction, NetworkSide side)
 		{
 			if (DefaultWillInteract.Default(interaction, side) == false) return false;
+			
+			if (interaction.TargetObject == null) return false;
+			
 			if (Validations.IsReachableByRegisterTiles(
 					interaction.Performer.RegisterTile(),
 					interaction.UsedObject.RegisterTile(),
 					side == NetworkSide.Server) == false) return false;
+			
+			if (Validations.IsReachableByRegisterTiles(
+				interaction.Performer.RegisterTile(),
+				interaction.TargetObject.RegisterTile(),
```

Comment:
I assume it's checking the object and the player  and the player and the bin

Reply:
yup